README.md

```diff
@@ -36,7 +36,7 @@ Provider specific code is being developed in separate repositories, some of whic
 are also sponsored by SIG-cluster-lifecycle:
 
   * AWS, https://github.com/kubernetes-sigs/cluster-api-provider-aws
-  * AWS/Openshift, https://github.com/openshift/cluster-operator
+  * AWS/Openshift, https://github.com/openshift/machine-api-operator/
```

Comment:
1) Does the machine api operator also based on cluster-api? 
2) Can you remove the last `/`?

Reply:
Yes, the machine api operator is based off of a subset of cluster-api (specifically Machines).

That said, I think it is more properly listed under API adoption (where it is already listed), so if the cluster-operator is now deprecated, it might be better to just remove it from this list.

Reply:
I found `https://github.com/openshift/machine-api-operator/` was already in https://github.com/kubernetes-sigs/cluster-api#api-adoption , so we can delete the `cluster-operator` from here.

Reply:
yes, I noticed that just now as well, thanks for the pointing and I will make changes